README.md

```diff
@@ -1,2 +1,161 @@
-# Portfolio-Project
-A webpage for my portfolio profile
+<h1 id = 'top'><b>Portfolio Project</b></h1>
+
+[![Status badge](https://img.shields.io/badge/CONTRIBUTORS-0-red.svg)](https://shields.io/) 
+[![Status badge](https://img.shields.io/badge/FORKS-0-yellow.svg?style=plastic&logo=appveyor)](https://shields.io/) 
+[![Status badge](https://img.shields.io/badge/REVIEWS-0-<red>.svg?style=plastic&logo=appveyor)](https://shields.io/) 
+[![Status badge](https://img.shields.io/badge/STARS-0-red.svg?style=plastic&logo=appveyor)](https://shields.io/) 
+[![Status badge](https://img.shields.io/badge/ISSUES-0-red.svg?style=plastic&logo=appveyor)](https://shields.io/) 
+[![Status badge](https://img.shields.io/badge/LICENSE-0-blue.svg?style=plastic&logo=appveyor)](https://shields.io/) 
```

Comment:
- Kindly use only the home page of your website, it will help reduce the the number of images on the docs and add more taste to proceed and use the link.

Reply:
Ok sir